src/components/Admin/GameLoader.jsx

```diff
@@ -0,0 +1,136 @@
+import { useTranslation } from "react-i18next";
+import "@/i18n/i18n";
+import { handleCsvFile, handleJsonFile } from "@/lib/utils";
+
+function isValidFileType(file, allowedTypes) {
+  const fileName = file.name.toLowerCase();
+  const fileExtension = fileName.split(".").pop();
+
+  if (!allowedTypes[fileExtension]) {
+    return false;
+  }
+
+  const mimePattern = allowedTypes[fileExtension].pattern;
+  return mimePattern.test(file.type);
+}
+
+const GameLoader = ({
+  gameSelector,
+  send,
+  setError,
+  setCsvFileUpload,
+  setCsvFileUploadText
+}) => {
+  const { i18n, t } = useTranslation();
+  return (
+    <div className="flex flex-col rounded  border-2">
+      <div className="flex translate-y-3 flex-row  items-center justify-center space-x-5 p-2">
+        {gameSelector.length > 0 ? (
+          <div>
+            <select
+              id="gameSelector"
+              defaultValue={""}
+              className="rounded border-2 bg-secondary-500 text-foreground"
+              onChange={(e) => {
+                send({
+                  action: "load_game",
+                  file: e.target.value,
+                  lang: i18n.language,
+                });
+              }}
+            >
+              <option disabled value="">
+                {t("Select question set")}
+              </option>
+              {gameSelector.map((value, index) => (
+                <option key={`set-${index}`} value={value}>
+                  {value.replace(".json", "")}
+                </option>
+              ))}
+            </select>
+          </div>
+        ) : null}
+        <div id="gamePickerFileUploadButton">
+          <label htmlFor="gamePickerFileUpload">
+            <div className="flex flow-row space-x-2 items-center">
+              <div className="w-6">
+                <svg
+                  className="cursor-pointer fill-current text-secondary-900 hover:text-secondary-500"
+                  viewBox="0 0 384 512"
+                >
+                  <path d="M224 136V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zm65.18 216.01H224v80c0 8.84-7.16 16-16 16h-32c-8.84 0-16-7.16-16-16v-80H94.82c-14.28 0-21.41-17.29-11.27-27.36l96.42-95.7c6.65-6.61 17.39-6.61 24.04 0l96.42 95.7c10.15 10.07 3.03 27.36-11.25 27.36zM377 105L279.1 7c-4.5-4.5-10.6-7-17-7H256v128h128v-6.1c0-6.3-2.5-12.4-7-16.9z" />
+                </svg>
+              </div>
+              <div>
+                <p className="text-secondary-900 text-s">{t("upload game")}</p>
+                <p className="text-secondary-900 text-xs">.json, .csv</p>
```

Comment:
```suggestion
                <p className="text-xs text-secondary-900">.json, .csv</p>
```
eslint tailwind classname order